### PR TITLE
CLICK HERE FOR REDTEXT

### PR DESCRIPTION
**Pull request by jughu**
LETS YOU BUY A GREENTEXT BOOK A CHALLENGE WHICH MARKS YOU FUCKING VALID FOR EVERYONE AND YOU NEED TO DO YOUR OBJECTIVES TOO AND YOU NEED TO GET THE BOOK TO ESCAPE AAAA!!! THE ULTIMATE CHALLENGE!!!! 

:cl: Hyena
add: The greentext challenge
/:cl:

[]: # (Please add a short description of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding:)the ultimate challenge for taitortots

### CODE REVIEW

**Comment by kevinz000** [comment]
So basically this is one click to send the station into chaos?
No.

**Comment by jughu** [comment]
its only a book that gives greentext :^)

**Comment by jughu** [comment]
and besides theres worser things that are one click and station chaos

**Comment by GunHog** [comment]
I would be interested to see it test-merged.

File: code/modules/uplink/uplink_item.dm [hunk 1]
```diff
@@ -1223,6 +1223,18 @@ var/list/uplink_items = list() // Global list so we only initialize this once.
 	cost = 20
 	surplus = 0
 	restricted_roles = list("Assistant")
+	
+// Challenges!
+/datum/uplink_item/Challenge
```
Comment (Cyberboss) [inline code/modules/uplink/uplink_item.dm]: One is capitalized, the other isn't

**Review by Cyberboss — CHANGES_REQUESTED** [review]

File: code/modules/uplink/uplink_item.dm (hunk 1 above)
Comment (GunHog) [inline code/modules/uplink/uplink_item.dm]: You did test this, right?????

File: code/modules/uplink/uplink_item.dm [hunk 2]
```diff
@@ -1223,6 +1223,18 @@ var/list/uplink_items = list() // Global list so we only initialize this once.
...
 	restricted_roles = list("Assistant")
+	
+// Challenges!
+/datum/uplink_item/challenge
+	category = "Challenges"
+	surplus = 0
+	
+/datum/uplink_item/challenge/greentext
+	name = "Greentext"
+	desc = "A book that will include your greentext that you have to carry to the shuttle, you also will be marked with a green color and other people will try steal your greentext!"
```
Comment (ShadowDeath6) [inline code/modules/uplink/uplink_item.dm]: This kills the language

Comment (jughu) [inline code/modules/uplink/uplink_item.dm]: I had a D on english ok im orginally dutch so pls write a grammar correctly lin ok

**Comment by jughu** [comment]
@anonmare probaly

**Comment by Cyberboss** [comment]
Greentext doesn't override your other objectives so this is worthless

On Mar 1, 2017 1:39 PM, "jughu" <notifications@github.com> wrote:

@anonmare <https://github.com/anonmare> probaly

—
You are receiving this because you commented.
Reply to this email directly, view it on GitHub
<https://github.com/tgstation/tgstation/pull/24587#issuecomment-283429295>,
or mute the thread

**Comment by jughu** [comment]
Auctally it adds a objective which makes it harder to greentext

**Comment by Cyberboss** [comment]
My point exactly

**Comment by jughu** [comment]
But as i said it was a challenge to make greentexting harder

**Comment by TehZombehz** [comment]
Further enforcing the concept of valids and how you should desire them is bad.
However, we've practically accepted it at this point so, I guess, why not abuse it? Nice meme.

**Comment by Shapsy** [comment]
Turning a rare, obscure meme into something that happens every round is dumb

**Comment by kevinz000** [comment]
It's not even just that it's a meme it's the fact that everyone will drop want they're doing and try to get it by lynching the current carrier. It's literally one click send station into a FFA and antags already have enough destructive items.

**Comment by PKPenguin321** [comment]
I can't believe nobody here has pointed out yet that touching the greentext traitors you
this will basically turn the whole station to traitors every round

**Comment by ExcessiveUseOfCobblestone** [comment]
Still a better love story than Romerol

**Comment by Cheridan** [comment]
i agree with above

**Comment by Cyberboss** [comment]
Ban @jughu PRs when?

**Comment by Arianya** [comment]
@Cyberboss Ironically he has a better merge vs closed ratio then goof.

**Pull request closed without merging**